source/tags/PlaceholderSvg/PlaceholderSvg.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PlaceholderSvg from './PlaceholderSvg';
+
+export default [{
+  name: "default",
+  component: (
+    <PlaceholderSvg width={600} height='300' />
```

Comment:
@krambuhl Sorry, which is which here? `height` and `width` are `props`, not HTML attributes, so shouldn't that be a single quote? And `name` is in the `new-tag` template with double quotes, so I'm guessing you don't mean that? My quote-blindness is getting the better of me here!